test/functional/options/winfixbuf_spec.lua

```diff
@@ -52,4 +52,24 @@ describe("Nvim API calls with 'winfixbuf'", function()
 
     assert(results == false)
   end)
+
+  it('Calling vim.api.nvim_win_set_buf on current buffer with `winfixbuf`', function()
+    local results = exec_lua([[
+      vim.wo.winfixbuf = true
+      local results, _ = pcall(vim.api.nvim_win_set_buf, 0, vim.api.nvim_get_current_buf())
+      return results
```

Comment:
this is mislelading because `results` actually contains the status, not the result. The usual convention is to name this `ok` or `status`. Let's do that for all tests in this file.

```suggestion
      local ok, _ = pcall(vim.api.nvim_win_set_buf, 0, vim.api.nvim_get_current_buf())
      return ok
```